container-engine-lib/lib/backend_impls/docker/docker_kurtosis_backend/engine_functions/create_engine.go

```diff
@@ -107,6 +108,7 @@ func CreateEngine(
 		logsAggregatorContainer,
 		defaultHttpLogsAggregatorPortNum,
 		sinks,
+		shouldTurnOffPersistentVolumeLogsCollection,
```

Comment:
Huh now I'm confused - why is DockerKurtosisBackend even exposing CreateLogsAggregator if the engine (which is the only thing that creates it to my knowledge) is bypassing it and going directly to `logs_aggregator_functions.CreateLogsAggregator`? I'd expect that the engine would use `DockerKurtosisBackend.CreateLogsAggregator` or it wouldn't exist; as-is, there are now two ways to create a logs aggregator and I think I'd be confused which I should be using.

Reply:
ye - tbh KurtosisBackend shouldn't expose `CreateLogsAggregator` since its not being used by anyone - but I'm going to leave that for a different change.